api/src/main/resources/MetadataTerm.hbm.xml

```diff
@@ -26,9 +26,8 @@ Copyright (C) OpenMRS, LLC.  All Rights Reserved.
 			<column name="metadata_source_id" />
 		</many-to-one>
 		<property name="code" type="java.lang.String" column="code" length="255" not-null="true" />
-		<property name="metadataClass" type="java.lang.String" column="metadata_class" length="1024" not-null="true" />
-		<property name="metadataUuid" type="java.lang.String" column="metadata_uuid" length="38"
-				  not-null="true" />
+		<property name="metadataClass" type="java.lang.String" column="metadata_class" length="1024" not-null="false" />
```

Comment:
In addition to making it optional here you need to add a liquibase changeset in liquibase.xml.